angular-rails-app/app/controllers/owners_controller.rb

```diff
@@ -3,10 +3,15 @@ class OwnersController < ApplicationController
    before_action :set_owner, only: [:show]
 
   def index
-    @owners = Owner.all
+    if params[:name]
+       @owners = Owner.where("name LIKE (?)", "%#{params[:name]}%").limit(6)
```

Comment:
@chrishenry Actually, this is going to make things much more complicated.  Because I can't only do this change on the back end.  I would have to display either names or corporation names in the drop down box the front end, depending on what what matched - so we'd have to keep track of that somehow...   I think we should either limit the search to name only at this point in time, or redesign our schema altogether so that we have a corporation owner type that is separate from an individual owner type or employee of a corporation.  Thoughts?